pretixprint/app/src/main/AndroidManifest.xml

```diff
@@ -90,6 +90,14 @@
                 <action android:name="eu.pretix.pretixpos.print.PRINT_RECEIPT" />
             </intent-filter>
         </service>
+
+        <meta-data android:name="io.sentry.dsn" android:value="${SENTRY_DSN}" />
```

Comment:
What will happen if SENTRY_DSN is empty? Will sentry be disabled (like before) or crash?

This is important e.g. for the F-Droid builds

Reply:
https://docs.sentry.io/platforms/android/configuration/options/#dsn
> If this value is not provided, the SDK will try to read it from the SENTRY_DSN environment variable. If that variable also does not exist, the SDK will just not send any events.

just compiled it without setting `sentry.dsn` in local.properties, it rightfully disables sentry :)

if you add `<meta-data android:name="io.sentry.debug" android:value="true" />` to the manifest, it also tells us this:
> D/Sentry: DSN is empty, disabling sentry-android